### PR TITLE
Add toolbar for ShippingLabelAddressSuggestionFragment

### PR DESCRIPTION
**Pull request by AnirudhBhat**
Part of #10570 

### Description

This PR introduces individual toolbars for ShippingLabelAddressSuggestionFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Create a shipping label CTA -> Ship from -> Address suggestion validation) screen and ensure the toolbar and its action work as expected on both phone and tablet.

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelAddressSuggestionFragment.kt [hunk 1]
```diff
@@ -63,11 +64,22 @@ class ShippingLabelAddressSuggestionFragment :
...
 
         _binding = FragmentShippingLabelAddressSuggestionBinding.bind(view)
-
+        setupToolbar()
         initializeViewModel()
         initializeViews()
     }
 
+    private fun setupToolbar() {
+//        binding.toolbar.title = getString(screenTitle)
```
Comment (samiuelson) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelAddressSuggestionFragment.kt]: ❓ What about the toolbar's title? If we don't show it, we should remove the `private var screenTitle` property.

There is also the `override fun getFragmentTitle()` function that looks redundant now.

**Review by samiuelson — APPROVED** [review]
Looks good overall. However, I left one comment with improvement ideas. Let me know what you think.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelAddressSuggestionFragment.kt (hunk 1 above)
Comment (AnirudhBhat) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelAddressSuggestionFragment.kt]: Yup. Removed it here: https://github.com/woocommerce/woocommerce-android/pull/10752/commits/12edd4a4f80771bd61de9014b4023fc41f26e1f5 https://github.com/woocommerce/woocommerce-android/pull/10752/commits/ca4639784a4bed2f5d7e1394ba29453e9744dc13

**Pull request merged**